src/cts/src/CMakeLists.txt

```
@@ -80,3 +80,28 @@ messages(
   TARGET cts
   OUTPUT_DIR ..
 )
+
+if (Python3_FOUND AND BUILD_PYTHON)
+  swig_lib(NAME          cts_py
+           NAMESPACE     cts
+           LANGUAGE      python
+           I_FILE        TritonCTS-py.i
+           SWIG_INCLUDES ${PROJECT_SOURCE_DIR}/include/cts
```

```suggestion
           SWIG_INCLUDES ${PROJECT_SOURCE_DIR}/../include/cts
```
This works on macOS. Why Linux is more forgiving I do not know.

---

Good catch Vitor, thanks! And you say it is strange that the Linux ones work, but it is also strange that the Mac one just uses the wrong file.  In both cases it should error out with a file not found. I guess there must be some default that is different on the two platforms?

---

Its totally mysterious but I don't want to spend the time to figure it out.

---

Agreed.